footer.php

```diff
@@ -1,8 +1,50 @@
-  <footer>
-    <p>This is my footer.</p>
+</body>
+
+<footer>
+
+<div class="container-fluid text-center">
+  <div class="row align-items-center bg-secondary" style="footer">
+    <div class="col-sm-2">
+      <img src="<?php echo get_template_directory_uri() . '/assets/images/ClimateHubLogo.gif'; ?>" class="img-fluid border-0 chlogo"/>
+    </div>
+    <div class="col-sm-2">
+    <?php get_template_part( 'template-parts/footer_customfield' )?>

```

Comment:
I think you can remove this line.

Reply:
line 11? if so, i deleted it (will push changes in a few)

Reply:
Yeah that's the line.